plugins/trezor/qt_generic.py

```diff
@@ -188,13 +188,15 @@ def create_handler(self, window):
 
     @hook
     def receive_menu(self, menu, addrs, wallet):
-        if type(wallet) is not Standard_Wallet:
-            return
         keystore = wallet.get_keystore()
```

Comment:
This line is no longer needed as the loop below overwrites the value of `keystore`.